### PR TITLE
VxLAN Tunnel Counters and Rates implementation 

### PR DESCRIPTION
**Pull request by dgsudharsan**
#### Why I did it
To Enable tunnel counters

#### How I did it
Enabling flex counters

#### How to verify it
Using show command and unit tests

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

**Review by srj102 — APPROVED** [review]

File: dockers/docker-orchagent/enable_counters.py [hunk 1]
```diff
@@ -43,6 +45,7 @@ def enable_counters():
     enable_counter_group(db, 'QUEUE_WATERMARK')
     enable_counter_group(db, 'BUFFER_POOL_WATERMARK')
     enable_counter_group(db, 'PORT_BUFFER_DROP')
+    enable_counter_group(db, 'TUNNEL')
```
Comment (prsunny) [inline dockers/docker-orchagent/enable_counters.py:48]: Lets disable the counter by default and enable based on need

File: dockers/docker-orchagent/enable_counters.py [hunk 2]
```diff
@@ -29,6 +29,8 @@ def enable_rates():
     counters_db.set('COUNTERS_DB', 'RATES:PORT', 'PORT_ALPHA', DEFAULT_ALPHA)
     counters_db.set('COUNTERS_DB', 'RATES:RIF', 'RIF_SMOOTH_INTERVAL', DEFAULT_SMOOTH_INTERVAL)
     counters_db.set('COUNTERS_DB', 'RATES:RIF', 'RIF_ALPHA', DEFAULT_ALPHA)
+    counters_db.set('COUNTERS_DB', 'RATES:TUNNEL', 'TUNNEL_SMOOTH_INTERVAL', DEFAULT_SMOOTH_INTERVAL)
+    counters_db.set('COUNTERS_DB', 'RATES:TUNNEL', 'TUNNEL_ALPHA', DEFAULT_ALPHA)
```
Comment (prsunny) [inline dockers/docker-orchagent/enable_counters.py:33]: What is the interval value?

Comment (dgsudharsan) [reply dockers/docker-orchagent/enable_counters.py:33]: Smooth interval is used in rate calculate where the traffic within this time limit are given higher weightage for rate calculation (for e.g. in last 10 sec over the traffic received prior to it) . https://github.com/Azure/SONiC/blob/master/doc/rates-and-utilization/Rates_and_utilization_HLD.md

**Review by prsunny — APPROVED** [review]

**Pull request merged**